docs/cli-reference/dfx-nns.md

```diff
@@ -0,0 +1,119 @@
+
+Use the `dfx nns` subcommands to interact with the Network Nervous System.
+
+The basic syntax for running `dfx nns` commands is:
+
+``` bash
+dfx nns [subcommand] [flag]
+```
+
+Depending on the `dfx nns` subcommand you specify, additional arguments, options, and flags might apply. For reference information and examples that illustrate using `dfx nns` commands, select an appropriate command.
+
+| Command                             | Description                                                                   |
+|-------------------------------------|-------------------------------------------------------------------------------|
+| [`import`](#_dfx_nns_import)        | Adds the NNS canisters to the local dfx.json as remote canisters.             |
+| [`install`](#_dfx_nns_install)      | Deploys NNS canisters to the local dfx server                                 |
+| `help`                              | Displays usage information message for a specified subcommand.                |
+
+To view usage information for a specific subcommand, specify the subcommand and the `--help` flag. For example, to see usage information for `dfx nns install`, you can run the following command:
+
+``` bash
+$ dfx nns install --help
+```
+
+
+## dfx nns import
+
+Use the `dfx nns import` command to add the NNS canisters to the local `dfx.json`.  It also downloads the did files and sets the canister IDs of the NNS cansiters so that you can make API calls to NNS canisters.
+
+### Basic usage
+
+``` bash
+$ dfx nns import
+```
+
+### Flags
+
+You can use the following optional flags with the `dfx nns import` command.
+
+| Flag                | Description                                   |
+|---------------------|-----------------------------------------------|
+| `-h`, `--help`      | Displays usage information.                   |
+| `--network-mapping` | Renames networks when installing canister ids |
+| `-V`, `--version`   | Displays version information.                 |
+
+### Examples
+
+You can use the `dfx nns import` command to get did files and so query NNS canisters.
```

Comment:
```suggestion
You can use the `dfx nns import` command to get did files and to query NNS canisters.
```

Reply:
This is actually intentional; getting did files enables making API calls.